ui/src/app/cohort-search/tree-node/tree-node.component.tsx

```diff
@@ -347,8 +347,8 @@ export class TreeNode extends React.Component<TreeNodeProps, TreeNodeState> {
         <div style={hover ? {...styles.treeNodeContent, background: colors.light} : styles.treeNodeContent}
           onMouseEnter={() => this.setState({hover: true})}
           onMouseLeave={() => this.setState({hover: false})}>
-          {(selectable && (source === 'criteria' || node.subtype === 'QUESTION')) && <button style={styles.iconButton}>
-            {(hasAttributes && (source === 'criteria'))
+          {(selectable && (node.subtype !== 'ANSWER')) && <button style={styles.iconButton}>
```

Comment:
This fixes concept search but we still need answers to be selectable in cohort builder.
<img width="848" alt="Screen Shot 2020-10-26 at 3 48 56 PM" src="https://user-images.githubusercontent.com/40036095/97227151-11c28180-17a3-11eb-8c28-464f8d24c58f.png">

Reply:
Ahh sorry

Reply:
Done